prov/usnic/src/usdf_fabric.c

```diff
@@ -780,6 +781,10 @@ usdf_usnic_getinfo(struct fid_fabric *fabric, struct fi_usnic_info *uip)
 	fp = fab_ftou(fabric);
 	dap = fp->fab_dev_attrs;
 
+	if (version > FI_USNIC_INFO_VERSION) {
```

Comment:
@xuywang Should `FI_USNIC_INFO_VERSION` be renamed to `FI_EXT_USNIC_INFO_VERSION`?

I ask because if we're adding something usnic-specific into the `FI_` namespace, it seems like we should put `EXT` in it.